src/modules/identity/actions.ts

```diff
@@ -0,0 +1,27 @@
+import { action } from 'typesafe-actions'
+import { AuthIdentity } from '@dcl/crypto'
+
+// Generate identity
+
+export const GENERATE_IDENTITY_REQUEST = '[Request] Generate Identity'
```

Comment:
In theory, there's no need to keep this actions, as we won't be handling them anywhere after the refactor in the sites. Are you leaving them to be compatible with what's built today?

Reply:
exactly, leaving it here so it's compatible with how it's used nowadays. If it's not used in the future, I'll remove it.